### PR TITLE
Additional userrights for igrovyesistemywiki T3536

### PR DESCRIPTION
**Pull request by Amanda-Catherine**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -4701,6 +4701,32 @@
 				'review' => true,
 				'validate' => true,
 			),
+			'ceo' => array(
+				'bureaucrat',
+				'sysmag',
+				'trusted',
+			),
+			'UserType1' = array(
+				'UserType1' = true,
```
Comment (MacFan4000) [inline LocalSettings.php]: should be 'UserType1',

File: LocalSettings.php [hunk 2]
```diff
@@ -4701,6 +4701,32 @@
...
+			),
+			'UserType2' = array(
+				'UserType2' = true,
```
Comment (MacFan4000) [inline LocalSettings.php]: same issue here

File: LocalSettings.php [hunk 3]
```diff
@@ -4701,6 +4701,32 @@
...
+			),
+			'UserType3' = array(
+				'UserType3' = true,
```
Comment (MacFan4000) [inline LocalSettings.php]: and

File: LocalSettings.php [hunk 4]
```diff
@@ -4701,6 +4701,32 @@
...
+			),
+			'UserType4' = array(
+				'UserType4' = true,
```
Comment (MacFan4000) [inline LocalSettings.php]: and here

File: LocalSettings.php [hunk 5]
```diff
@@ -4701,6 +4701,32 @@
 				'review' => true,
 				'validate' => true,
 			),
+			'ceo' => array(
+				'bureaucrat',
```
Comment (The-Voidwalker) [inline LocalSettings.php]: missing `=> true`

File: LocalSettings.php [hunk 6]
```diff
@@ -4701,6 +4701,32 @@
...
+				'UserType2' = true,
+			),
+			'UserType3' = array(
+				'UserType3' = true,
+			),
+			'UserType4' = array(
+				'UserType4' = true,
+			),
+			'UserType5' = array(
+				'UserType5' = true,
```
Comment (MacFan4000) [inline LocalSettings.php]: and here

File: LocalSettings.php [hunk 7]
```diff
@@ -4701,6 +4701,32 @@
 				'review' => true,
 				'validate' => true,
 			),
+			'ceo' => array(
+				'bureaucrat',
+				'sysmag',
+				'trusted',
+			),
+			'UserType1' = array(
```
Comment (The-Voidwalker) [inline LocalSettings.php]: use `=>` instead of `=`

File: LocalSettings.php [hunk 8]
```diff
@@ -4701,6 +4701,32 @@
...
+				'UserType3' = true,
+			),
+			'UserType4' = array(
+				'UserType4' = true,
+			),
+			'UserType5' = array(
+				'UserType5' = true,
+			),
+			'UserType6' = array(
+				'UserType6' = true,
```
Comment (MacFan4000) [inline LocalSettings.php]: and here

File: LocalSettings.php [hunk 9]
```diff
@@ -4701,6 +4701,32 @@
...
+			),
+			'UserType7' = array(
+				'UserType7' = true,
```
Comment (MacFan4000) [inline LocalSettings.php]: and here

File: LocalSettings.php [hunk 10]
```diff
@@ -4701,6 +4701,32 @@
...
+				'UserType1' => true,
+			),
+			'UserType2' => array(
+				'UserType2' => true,
+			),
+			'UserType3' => array(
+				'UserType3' => true,
+			),
+			'UserType4' => array(
+				'UserType4' = true,
```
Comment (The-Voidwalker) [inline LocalSettings.php]: Looks like you missed this one

File: LocalSettings.php [hunk 11]
```diff
@@ -4701,6 +4701,32 @@
 				'review' => true,
 				'validate' => true,
 			),
+			'ceo' => array(
+				'bureaucrat' => true,
```
Comment (MacFan4000) [inline LocalSettings.php]: missing a comma

File: LocalSettings.php [hunk 12]
```diff
@@ -4701,6 +4701,32 @@
...
+				'sysmag', => true,
+				'trusted', => true,
```
Comment (The-Voidwalker) [inline LocalSettings.php]: extra comma on these two lines

**Comment by Amanda-Catherine** [comment]
Yay, it finally passed!

**Pull request merged**